Packages/com.unity.inputsystem/InputSystem/Controls/InputControl.cs

```diff
@@ -134,6 +134,11 @@ public virtual object valueAsObject
             get { return null; }
         }
 
+        public virtual object previousValueAsObject

```

Comment:
If we add this, `previousValueAsObject` will have to be implemented at the `InputDevice` level and these things will need tests.

Reply:
because it's not actually a previousValue, I'll revert this change